tests/partitions_test.py

```diff
@@ -148,6 +150,56 @@ def test_isolate_and_reconnect_primary(network, args, **kwargs):
         assert status == TxStatus.Invalid, r
 
 
+@reqs.description("New joiner helps liveness")
+def test_new_joiner_helps_liveness(network, args):
+    primary, backups = network.find_nodes()
+
+    # Issue some transactions, so there is a ledger history that a new node must receive
+    network.txs.issue(network, number_txs=10)
+
+    # Remove a node, leaving the network frail
```

Comment:
Does this test only makes sense if there are 3 nodes in the network to start with? If so, this could be expressed as a requirement.